spec/comm.mediawiki

```diff
@@ -191,9 +191,23 @@ of their penalization.
 |-
 | matches || &#91;object&#93; || list of [[orders.mediawiki/#Match_negotiation|Match objects]]
 |-
-| penalty || object || a [[community.mediawiki/#Penalty_Object|Penalty Object]]. omitted if in good standing
+| order statuses || <nowiki>[object]</nowiki> || list of <code>Order Status Object</code>
 |-
-| sig     || string || hex-encoded server's signature of the serialized connection data
+| score || int || The user's score. The math behind this is currently being adjusted
+|-
+| suspended || bool || true if suspended. will be deprecated in a future version
+|-
+| sig || string || hex-encoded server's signature of the serialized connection data
+|}
+
+'''Order Status Object'''
+
+{|
+| field || type || description
```

Comment:
Should be headers like the other tables with `!` instead of `|`

Reply:
Ok https://github.com/decred/dcrdex/compare/76f17fac855decc14288d37c0865e68329fed2d8..b2ea39a42dca73ae454cdadd3fa003d01b02de5e